src/util/system.cpp

```diff
@@ -1128,10 +1128,22 @@ fs::path GetSpecialFolderPath(int nFolder, bool fCreate)
 void runCommand(const std::string& strCommand)
 {
     if (strCommand.empty()) return;
+    int nErr = 0;
 #ifndef WIN32
-    int nErr = ::system(strCommand.c_str());
+    nErr = ::system(strCommand.c_str());
 #else
-    int nErr = ::_wsystem(std::wstring_convert<std::codecvt_utf8_utf16<wchar_t>,wchar_t>().from_bytes(strCommand).c_str());
+    STARTUPINFOA info;
+    PROCESS_INFORMATION processInfo;
+    ZeroMemory(&info, sizeof(info));
+    info.cb = sizeof(info);
+    ZeroMemory(&processInfo, sizeof(processInfo));
```

Comment:
Is there a need to zero this?

Reply:
I had never used the windows API before this, so I sticked to the documentation example : https://docs.microsoft.com/en-us/windows/desktop/ProcThread/creating-processes